src/components/cards/TripCard/TripCard.jsx

```diff
@@ -61,7 +61,7 @@ const TripCard = ({
           <Date startDate={startDate} endDate={endDate} isSmall />
         </div>
         <div className={styles.content}>
-          <Link to={tripURL} className={styles.link}>
+          <SbLink link={{ url: tripURL }} classes={styles.link}>
```

Comment:
```suggestion
          <SbLink link={{ url: tripURL }} className={styles.link}>
```

Not for today, but for future us, it would be nice to get rid of this older convention and get into `className` everywhere again.